src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,28 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int pouwer;
```

Comment:
and rename related methods. please
```suggestion
    private int power;
```